app/main.py

```diff
@@ -1,3 +1,69 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
```

Comment:
The type hint for `km` is `int`, but methods like `__truediv__` create `Distance` objects with floating-point values for `km` (e.g., `Distance(2.85)`). To make the type hints consistent with the class's behavior, it's better to allow `km` to be an `int` or a `float` in the initializer.

```suggestion
    def __init__(self, km: int | float) -> None:
```